### PR TITLE
[pull-request:merge] Add a closing comment after closing PR

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

**TODO:**
- [x] Add tests.

### CODE REVIEW

**Comment by sstok** [comment]
Nice 😊 👍

**Comment by phansys** [comment]
Do you know how can I add a test for this feature? I didn't found any similar test in the current suite.

File: src/Command/PullRequest/PullRequestMergeCommand.php [hunk 1]
```diff
@@ -364,4 +364,38 @@ function ($value) {
 
         return $prType;
     }
+
+    /**
+     * Adds a note on the closed PR pointing to the merge commit.
```
Comment (cordoval) [inline src/Command/PullRequest/PullRequestMergeCommand.php]: i would otherwise remove this docblock, it does not add anything new or useful

File: tests/Fixtures/Adapter/TestAdapter.php [hunk 2]
```diff
@@ -249,16 +249,31 @@ public function closeIssue($id)
...
+            'user' => 'phansys',
+            'created_at' => new \DateTime('1969-12-31T10:00:00+0100'),
+            'updated_at' => new \DateTime('1969-12-31T10:00:00+0100'),
+        ];
+
+        return $url;
     }
 
     /**
      * {@inheritdoc}
```
Comment (cordoval) [inline tests/Fixtures/Adapter/TestAdapter.php]: even these blocks are useless, i don't think even phpstorm drills on them to display you their parent's signature. But these we are tolerating so far. 👍

Comment (phansys) [inline tests/Fixtures/Adapter/TestAdapter.php]: You mean the `{@inheritdoc}` docblock? I think they are useless for majority of IDEs I know, but they are useful for documentation tools.
BTW, I didn't add this particular block here, but I've used it in other methods.

Comment (cordoval) [inline tests/Fixtures/Adapter/TestAdapter.php]: i know 👍 
Gush will never be documented in this way because it is not a library neither a framework. For this reason i am strongly considering removing those.

File: tests/Command/PullRequest/PullRequestMergeCommandTest.php [hunk 3]
```diff
@@ -94,6 +94,18 @@ class PullRequestMergeCommandTest extends CommandTestCase
 New-target: %s/%s (was "%s")
 OET;
 
+    const MERGE_NOTE_SWITCHED_BASE_AND_CLOSED = <<<OET
```
Comment (cordoval) [inline tests/Command/PullRequest/PullRequestMergeCommandTest.php]: what was OET? i forgot

Comment (phansys) [inline tests/Command/PullRequest/PullRequestMergeCommandTest.php]: I really don't know, just used the same heredoc delimiter that already were using the other consts. BTW, I'd like to update the usages of heredoc with nowdoc in another commit.

Comment (cordoval) [inline tests/Command/PullRequest/PullRequestMergeCommandTest.php]: send another PR, i am not sure what you mean but we will see there

Comment (phansys) [inline tests/Command/PullRequest/PullRequestMergeCommandTest.php]: @cordoval: http://php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc

Comment (cordoval) [inline tests/Command/PullRequest/PullRequestMergeCommandTest.php]: i see nice
